Description: Installations outside /var/www are disabled by default
 This friendly report helped me out
 http://dabase.com/blog/AH01630:_client_denied_by_server_configuration/
--- a/tools/make_project
+++ b/tools/make_project
@@ -327,6 +327,7 @@
     <Directory "%(proot)s/html">
         Options Indexes FollowSymlinks MultiViews
         AllowOverride AuthConfig
+	Require all granted
         Order allow,deny
         Allow from all
     </Directory>
@@ -334,6 +335,7 @@
     <Directory "%(proot)s/cgi-bin">
         Options ExecCGI
         AllowOverride AuthConfig
+	Require all granted
         Order allow,deny
         Allow from all
     </Directory>
